src/unk_0202631C.c

```diff
@@ -65,13 +65,13 @@ static inline void CheckPokedexIntegrity(const PokedexData *param0)
     GF_ASSERT(param0->magic == 3203386110);
 }
 
-static BOOL sub_02026344(u16 param0)
+static BOOL Pokedex_SpeciesOutOfBounds(u16 species)
```

Comment:
```suggestion
static BOOL IsSpeciesOutOfBounds(u16 species)
```

Breaking the rules on module-prefixing is totally fine for local functions. The biggest reason for module-prefixing in the repo is to avoid linker collisions, but that's not really a concern for local functions.